## Protein Structure Estimation from Data Mining

### Project Description

#### Summary

Motivated by the need for fast, reliable methods of assessing protein structure in the era of big genomics, this continuing effort seeks to provide data-derived tools for high-throughput structural characterization of proteins directly from traditional (e.g., HSQC) nuclear magnetic resonance (NMR) spectra, prior to resonance assignment. By correlating the NMR chemical shifts and structure content of proteins of known conformation, these tools allow accurate, real-time estimation of secondary structure in previously uncharacterized proteins.

Projects contributing to this effort have, e.g., 1) demonstrated an empirical correlation between the averaged chemical shift (ACS) of protein backbone nuclei and observed secondary structure (&alpha;&#8209;helix and &beta;&#8209;strand) content as a fraction of overall conformation;<sup>1-3</sup> 2) confirmed these results by applying rigorous statistical methods (see below) to establish further correlations between ACS and protein structural classes;<sup>3-5</sup> and 3) extended the latter using a neural network strategy.<sup>6</sup>  

Ongoing projects expand on previous deep learning approaches, and apply alternative machine learning strategies, including logistic regression, to revised datasets.  

<img width="75%" height="75%" src="/images/Prot-struct-classes.png?raw=true"/>

#### Impacts

... "the observed correlation between ACS and SSC can be used to monitor structural changes in real time, such as in protein folding experiments, to detect large-scale structural changes in complex formation and to identify initial protein folds in high throughput proteomics applications." ... \[cf., our similar statement\] ...

#### Methods

*Protein Structural Class Identification from Averaged NMR Chemical Shifts: Statistical Basis*\[Refs\]

\[... Strategy / approach / data/databases / KS test ... 

... Languages / platforms ... 

... Link to GH project / code ...

Objective / Analytic Approach/Strategy / Data / Model / Evaluation:\]

I. Objective

...

II. Analytic Approach

...

...

#### GitHub Project Links

...

#### References
<!-- ... [Link](https://.../) ... -->
1. Sibley, A.B., Cosman, M. and Krishnan, V.V. (2003) An empirical
correlation between secondary structure content and averaged
chemical shifts in proteins. *Biophys. J.* **84**, 1223–1227.

2. Mielke, S.P. and Krishnan, V.V. (2005) Estimation of protein secondary structure content directly from NMR spectra using an improved empirical correlation with averaged chemical shift. *Journal of Structural and Functional Genomics* **6**, 281–285.

3. Mielke, S.P. and Krishnan, V.V. (2008) Characterization of protein secondary structure from NMR chemical shifts. *Progress in NMR Spectroscopy* **54**, 141–165.

4. Mielke, S.P. and Krishnan, V.V. (2003) Protein structural class identification directly from NMR spectra using averaged chemical shifts. *Bioinformatics* **19**, 2054–2064.

5. Mielke, S.P. and Krishnan, V.V. (2003) Chemical shift-based identification of protein structural classes. *Biophysical Journal* **84**, 460A. 

6. Kumar, A.V., Rehana, F.M.A., Cao, Y. and Krishnan, V.V. (2015) Application of data mining tools for classification of protein structural class from residue based averaged NMR chemical shifts. *Biochimica et Biophysica Acta (BBA) - Proteins and Proteomics* **1854**, 1545-1552.


